executor/adapter.go

```diff
@@ -360,6 +360,14 @@ func (a *ExecStmt) buildExecutor(ctx context.Context) (Executor, error) {
 func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
 	cfg := config.GetGlobalConfig()
 	costTime := time.Since(a.startTime)
+	threshold := time.Duration(cfg.Log.SlowThreshold) * time.Millisecond
+	level := log.GetLevel()
+	if costTime < threshold && level < log.DebugLevel {
```

Comment:
I think using `costTime < threshold` is enough.

Reply:
If only costTime < threshold here, when level is Debug, the query log will miss.